app/views/users/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>user <%= @id %></h1>
+<div class="container my-3">
+  <div class="row">
+  <div class="col-3">
+  <%= image_tag @user[:photo], :size => "150x150", class:"rounded" %>
+  </div>
+  <div class="border rounded col-9 d-flex align-items-center justify-content-between">
+  <h3><%= @user[:name] %></h3>
+  <span>Number of posts:<%= @user[:postsCounter] %></span>
+  </div>
+  </div>
+</div>
+<div>
+<div class="container border rounded my-3">
+  <h4> Bio </h4>
+  <p><%= @user[:bio] %></p>
+  </div>
+</div>
+<% @posts.each do |post| %>
+<div class="container border rounded my-3">
+<h4> <%= post[:title]%> </h4>
+<p> <%= post[:text]%> </p>
+<div class="d-flex justify-content-end">
+<span>Comments: x</span>
+<span>Likes: y </span>
+</div>
+</div>
+<% end %>
+<div class=" container col-12 text-center">
+<button class="border rounded">See all posts</button>
```

Comment:
Button does not work for `see all posts`

- [ ] Please use the right tag so when a user clicks `see all posts` they can view it.